cfme/tests/automate/test_common_methods.py

```diff
@@ -250,3 +253,71 @@ def test_service_retirement_from_automate_method(request, generic_catalog_item,
         description=f"Service Retire for: {generic_catalog_item.name}")
     wait_for(lambda: retire_request.request_state == "finished", fail_func=retire_request.reload,
              timeout=180, delay=10)
+
+
+@pytest.fixture
+def set_root_tenant_quota(request, appliance):
+    field, value = request.param
+    root_tenant = appliance.collections.tenants.get_root_tenant()
+    root_tenant.set_quota(**{'{}_cb'.format(field): True, field: value})
+    yield
+    root_tenant.set_quota(**{'{}_cb'.format(field): False})
+
+
+@pytest.mark.tier(3)
+@pytest.mark.usefixtures("setup_provider")
+@pytest.mark.meta(automates=[1334318])
+@pytest.mark.provider([VMwareProvider], override=True)
+@pytest.mark.parametrize(
+    ['set_root_tenant_quota'],
+    [
+        [('memory', '1000')],
+    ],
+    indirect=['set_root_tenant_quota'],
+    ids=['memory']
+)
+def test_automate_quota_units(request, set_root_tenant_quota, provisioning, provider, appliance):
+    """
+    Bugzilla:
+        1334318
+
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: low
+        initialEstimate: 1/4h
+        tags: automate
+    """
+    vm_name = random_vm_name(context='quota')
+
+    prov_data = {
+        "catalog": {'vm_name': vm_name},
+        "environment": {'automatic_placement': True},
+        "network": {'vlan': partial_match(provisioning['vlan'])},
+        'hardware': {'memory': '1024'},
+    }
+
+    @request.addfinalizer
+    def _finalize():
+        collection = appliance.provider_based_collection(provider)
+        vm_obj = collection.instantiate(vm_name, provider, provisioning["template"])
+        try:
+            vm_obj.cleanup_on_provider()
+        except Exception:
+            logger.warning('Failed deleting VM from provider: %s', vm_name)
+
+    with LogValidator(
+            "/var/www/miq/vmdb/log/automation.log",
+            matched_patterns=[f'.*Getting Tenant Quota Values for:.*.memory=>1073741824000.*'],
+    ).waiting(timeout=120):
+        # Provisioning VM via lifecycle
+        do_vm_provisioning(appliance, template_name=provisioning["template"], provider=provider,
+                           vm_name=vm_name, provisioning_data=prov_data, wait=False, request=None)
+
+        # nav to requests page to check quota validation
+        request_description = f'Provision from [{provisioning["template"]}] to [{vm_name}]'
```

Comment:
I've sometimes seen that request description changes for different appliance versions. If that happens with you, you might want to find a solution to that.

Reply:
Actually, I have not seen this type of issue with request description of VM provisioning. 
Please let me know the issue related to it and solution you have performed.

Reply:
I faced similar issue while dealing with VM retirement request.